dist-raw/node-cjs-loader-utils.js

```diff
@@ -0,0 +1,118 @@
+// copied from https://github.com/nodejs/node/blob/master/lib/internal/modules/cjs/loader.js
```

Comment:
Is this whole file copied or which parts should be reviewed? Curious on things like we read JSON ourselves instead of relying on the `require` cache.

Reply:
Good point, this is actually coming from several files in node's source.  I added comments to each item in this file with a permalink to the span of node's source code on github.